### ISSUE TITLE
[NEW FEATURE] Multi-level menu for plots

### ISSUE THREAD

**Issue by dharabor** [bshen95/MAPF-benchmark-web#12]
Some plot displays show multiple camvases and are to hard to follow. Suggest to add multi-level menu when selecting the type of comparison (from the table controls). Then only one plot is shown at any time.

**Comment by bshen95** [comment]
This is fixed.

**Comment by dharabor** [comment]
Looks good. I suggest adding the word "By" in front of the second-level menus though. Like "Compare Algorithms" -> "By Domain"

I notice a bit of weirdness in the menus also; a bounding box is faintly visible around the menu item text. The click action only registers when the cursor is inside this area. It should register when the cursor is inside the row of the menu item. The issue affects menus at other levels as well.
